### ISSUE TITLE
Inline Operation - Document if it requires 'fetch' operation also

### ISSUE THREAD

**Issue by realtebo** [Laravel-Backpack/docs#489]
Page: https://backpackforlaravel.com/docs/6.x/crud-operation-inline-create

I am setting up field as

        CRUD::field([
                'name' => 'address',
                'inline_create' => true,
            ])
            ->type('relationship')
            ->label('Address')
            ->wrapper(['class' => 'form-group col-md-6'])
        ;

Note: i am omitting ajax

Anyway, this field is automatically a select 2 with ajax. 

This need fetch operation, I think. Because opening the select2, an ajax call to a non-existent route ../fetch/... is fired

Please, document it, (or explain me what am I doing wrong).

**Comment by karandatwani92** [comment]
Yes, you need to use the fetch operation. It's documented [here](https://backpackforlaravel.com/docs/6.x/crud-fields#load-entries-from-ajax-calls-using-the-fetch-operation-1) under the `relationship` field.

Hope this would get you through. Please remember to close the issue.

For more queries, this would be a better place https://github.com/Laravel-Backpack/community-forum/issues
Good day @realtebo

**Comment by realtebo** [comment]
Thanks, but add a side note into the iline documentation becase if,as us, are not using the relationship field, you will not face this row of documentation.

**Comment by tabacitu** [comment]
> Anyway, this field is automatically a select 2 with ajax.

Hmm I don't think that's intentional. I think that's a bug, to be honest. If you ask me, you should be able to use `inline_create` with a non-ajax field. I don't see why not.

@pxpm let me do a TLDR for you (everybody correct me if I'm wrong here):
- @realtebo tried to add a relationship field
- as soon as he use `inline_create` relationship turned the field into an AJAX one
- but that triggered an error, because he didn't have the Fetch operation set up

Are you aware of this restriction @pxpm ? Did we choose to do this? Should we document or fix this?

**Comment by pxpm** [comment]
Like you rephrased looks like a restriction, but it's actually the "defaults" we choose at the time and I am still standing by them as I still think they are the most convenient.  

The field is indeed by default an ajax field and that's mentioned in the docs:
![image](https://github.com/Laravel-Backpack/docs/assets/7188159/2f2ee028-47e4-4e17-9bda-c863ff9d1982)

We even mention a Fetch caveat there:
![image](https://github.com/Laravel-Backpack/docs/assets/7188159/b752a424-1be4-4796-8468-1f287417f50c)


It's **NOT MANDATORY** to use FetchOperation, you can use `data_source => 'your.endpoint`.

**Comment by karandatwani92** [comment]
What I get after reading the conversations is, Info is already there but not in the highlights.

@tabacitu do you feel to do something here? It feels like you as the right person to take over this.
